app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
```

Comment:
Oh, download policy can only be set for yum repositories right? So we'd only want to set this if `self.yum?` ? And otherwise have it return nil?